prow/plugins/trigger/pull-request.go

```diff
@@ -180,6 +173,7 @@ func abortAllJobs(c Client, pr *github.PullRequest) error {
 		if job.Complete() {
 			continue
 		}
+		job.SetComplete()
```

Comment:
It is actually important that we do not do this, because Planks aborting ignores completed jobs.

Reply:
Thank you for catching that. Fixed (It seems I was using an old CRD for prowjobs, which required `completionTime` on `Aborted` state).